LICENSE.md

```diff
@@ -8,7 +8,7 @@ Version 2.0.2
 
 **Game**: Minecraft, the Minecraft server, and all associated software or derivatives, as defined by Mojang.
 
-**Mod**: Software which modifies the Game, in any form (source, compiled binary, etc).
+**The Mod**: Any copyrightable work licensed under this license, in any form (source, compiled binary, etc).
```

Comment:
Huh. I thought I caught all of those. It should be "the mod", because the license is intended to apply to any mod, not a specific mod. Eg, a modder says "my mod is licensed under $license", and the license says "the mod...various specific things".

It's possibly a personal preference, I'll put it on the list of "things to ask my lawyer".